lib/mimdb_web/templates/layout/app.html.eex

```diff
@@ -12,16 +12,9 @@
     <header>
       <section class="container">
         <nav role="navigation">
-          <ul>
-            <li><a href="https://hexdocs.pm/phoenix/overview.html">Get Started</a></li>
-            <%= if function_exported?(Routes, :live_dashboard_path, 2) do %>
-              <li><%= link "LiveDashboard", to: Routes.live_dashboard_path(@conn, :home) %></li>
-            <% end %>
-          </ul>
+          <%= render "_user_menu.html", assigns %>
         </nav>
-        <a href="https://phoenixframework.org/" class="phx-logo">
-          <img src="<%= Routes.static_path(@conn, "/images/phoenix.png") %>" alt="Phoenix Framework Logo"/>
-        </a>
+          <img src="<%= Routes.static_path(@conn, "/images/mimdb.png") %>" alt="mIMDB Logo"/>
```

Comment:
Should we change this to a link and choose a landing page? Not at all required via Story, but might be a nice addition. Or perhaps wait until Movies are done and use that as logical landing page?

Reply:
I was thinking Movies would be a natural landing page for the site.